Units/parser-c.r/macros.c.d/expected.tags

```diff
@@ -1,8 +1,9 @@
+DECL1	input.c	/^DECL1(foo); \/* gcc will accept this as function prototype (with some warnings) *\/$/;"	p	signature:(foo)
+DECL3	input.c	/^DECL3(x, y); \/* gcc will accept this as function prototype (with some warnings) *\/$/;"	p	signature:(x, y)
```

Comment:
(I have to look again this. I saw these expressions are used in many places in linux kernel. e.g.MODULE_INIT.
I have to see the impact on these area of new parser.)